app/services/ticker.py

```diff
@@ -0,0 +1,183 @@
+import asyncio
+from datetime import datetime
+from decimal import getcontext, InvalidOperation, DivisionByZero
+import logging
+from queue import Queue, Empty as QueueEmpty
+from web3 import Web3
+
+from ..app import App
+from ..src.erc20_token import ERC20Token
+
+logger = logging.getLogger('services.ticker')
+logger.setLevel(logging.DEBUG)
+
+getcontext().prec = 8
+
+# TODO: finally extract these into constants
+ZERO_ADDR = "0x0000000000000000000000000000000000000000"
+ZERO_ADDR_BYTES = Web3.toBytes(hexstr=ZERO_ADDR)
+
+tokens_queue = Queue()
+# TODO: Populate from our own DB
```

Comment:
Yeah, this is pretty important. Is there any way to import it from the main config for now?

Reply:
Not really.